src/ports/postgres/modules/knn/test/knn.sql_in

```diff
@@ -0,0 +1,6 @@
+m4_include(`SQLCommon.m4')
+/* -----------------------------------------------------------------------------
+ * Test knn.
+ *
+ * FIXME: Verify results
+ * -------------------------------------------------------------------------- */
```

Comment:
Please go ahead and add few test cases, that would
check if knn is actually running as expected. Use small toy
data sets to do the same. We can run install checks to
perform some basic sanity checks of the module.